docker/requirements_anndata.in

```diff
@@ -0,0 +1,2 @@
+# requirements.in packages for minimal ScPCA workflows
+anndata~=0.10
```

Comment:
Just noting that I think we need an image with scvi tools and anndata together. The process for classifying with CellAssign runs two scripts, the AnnData conversion and then CellAssign which would require scvi tools.

Reply:
This makes sense. I was kind of thinking we might still use the full image for that step: The only things removed would be Seurat and rmarkdown, which may not be too big. But maybe they are.